modules/two_factor_authentication/app/views/two_factor_authentication/my/backup_codes/show.html.erb

```diff
@@ -1,11 +1,13 @@
 <% html_title(t(:label_my_account), t('two_factor_authentication.backup_codes.plural')) -%>
 
-<h1>
-  <%= t('two_factor_authentication.backup_codes.plural') %>
-  <small>
-    <%= t('two_factor_authentication.backup_codes.your_codes', login: current_user.login, app_name: Setting.app_title) %>
-  </small>
-</h1>
+<%=
+  render(Primer::OpenProject::PageHeader.new) do |header|
+    header.with_title { "#{ t('two_factor_authentication.backup_codes.plural') } #{t('two_factor_authentication.backup_codes.your_codes', login: current_user.login, app_name: Setting.app_title)}" }
```

Comment:
<img width="400" alt="Bildschirmfoto 2024-08-13 um 11 42 30" src="https://github.com/user-attachments/assets/2af35387-5fce-444f-b0a7-1ab557fba3a8">

That looks weird. I think "Backup codes" would be enough in the title and the breadcrumb. As you are in the MyAccount section, users know for which user they are generating codes.